docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -217,24 +224,60 @@ Unlike agent-based models, microsimulations do not necessarily program agency in
-populations with differing characteristics.
+Like other microsimulation frameworks [[add example and citation]], Vivarium utilizes randomly generated numbers to incorporate stochastic uncertainty into our simulations. 
+However, unlike most other microsimulation frameworks, the IHME microsimulation platform additionally reduces variance between scenarios by appropriately controlling 
+stochastic uncertainty through a common-random-numbers approach. In this procedure, for each simulant, the same randomly generated number is used in both baseline and 
+intervention scenarios to dictate change or maintenance of health status across time steps. The only difference between scenarios, then, is the presence or absence of 
+health interventions.
+
+.. todo:: 
+  
+  Example of microsim that uses random numbers but not common random numbers? 
+
+
+Computational needs
++++++++++++++++++++
+
+To date, all Vivarium simulations require a super-computer (such as a high-performance cluster) to run. [[refer to previous section on details]] Other microsimulation frameworks may have prioritized computational ease, such as starsim...  
+
+.. todo:: 
+  
+    Elaborate on HPVsim's inclusion of ways to run simulations in parallel: 
+    - the “MultiSim” class for running, summarizing, and plotting an arbitrary collection of simulations; the “Scenario” class for running, summarizing, and plotting a collection of simulations that specifically differ according to their input parameters, including interventions; the “Sweep” class for running 2+ dimensional sweeps over parameter values and producing automated heatmaps of the resulting outcomes of interest. 
+    - All of these can be used on personal computers to run a small number of simulations in parallel, or to run very large numbers of simulations on cloudbased or HPC computing resources
+
+
+Type of event simulation
+++++++++++++++++++++++++
+
+The way simulants undergo events (i.e., move from one state to another) can be recorded in different ways, depending on microsimulation methods. These divergent methods are referred to as discrete time simulation (simulants progress at fixed time increments, and it is recorded if they experienced an event) and discrete event simulation (simulants
+move from event to event based on sampled timing of those events occuring). There are advantages and disadvantages related to each of these disparate methods of event simulation,
+which we elaborate upon below. 
+
+
+.. todo:: 
+
+  Discrete time simulation (continuous event): e.g. Vivarium (https://vivarium-research.readthedocs.io/en/latest/model_design/designing_vivarium_model/choosing_appropriate_time_step/index.html?highlight=choosing%20an%20appropriate#choosing-an-appropriate-time-step) 
+    
+  - Advantages: results in the same distribution as with discrete event 
+  - Disadvantages: staying in each state for the same amount of time, less computationally efficient
```

Comment:
I realize this is a todo, but we will probably want to soften this statement: we can get the time spent in states right, as long as our timestep isn't too small.

Reply:
This might be helpful to discuss this a bit more in the next doc session, but I'll try to finalize this section more before then.